android/src/main/java/com/sailthru/mobile/rnsdk/RNSailthruMobilePackage.java

```diff
@@ -20,7 +22,8 @@
 public class RNSailthruMobilePackage implements ReactPackage {
     protected ReactApplicationContext reactApplicationContext;
     protected boolean displayInAppNotifications = true;
-    private SailthruMobile sailthruMobile = new SailthruMobile();
+    @VisibleForTesting
+    SailthruMobile sailthruMobile = new SailthruMobile();
```

Comment:
my favourite way to do this is by adding a `default` visibility constructor that can be called in tests and make the public constructor create the object and call the former. This takes up fewer lines, though.